nginx/config.go

```diff
@@ -48,7 +48,8 @@ http {
 
 	client_max_body_size {{ $routerConfig.BodySize }};
 
-	set_real_ip_from {{ $routerConfig.ProxyRealIPCIDR }};
+	{{ range $realIPCIDR := $routerConfig.ProxyRealIPCIDRs }}set_real_ip_from {{ $realIPCIDR }};{{ end }}
+	real_ip_recursive on;
```

Comment:
Looks like some nicer formatting options made their way into templates last fall and I hadn't noticed since then:

https://github.com/golang/go/commit/e6ee26a03b79d0e8b658463bdb29349ca68e1460